src/code42cli/cmds/securitydata.py

```diff
@@ -196,33 +199,19 @@ def search(
     **kwargs
 ):
     """Search for file events."""
-    cursor = (
-        _get_file_event_cursor_store(state.profile.name) if use_checkpoint else None
-    )
-
-    handlers = ext.create_handlers(
-        state.sdk,
-        FileEventExtractor,
-        cursor,
+    print_events_decorator = print_events(format, include_all, SEARCH_DEFAULT_HEADER)
+    handlers = _extract_events(
+        state,
+        begin,
+        end,
+        advanced_query,
         use_checkpoint,
-        include_all,
-        output_format=format,
-        output_header=SEARCH_DEFAULT_HEADER,
+        saved_search,
+        or_query,
+        output_function=print_events_decorator,
```

Comment:
Shouldn't need an output function, only the handlers need to know how to output the events.

Create different handlers for `send-to` and have it use the SyslogLogger rather than echo... There will likely be logic sharing there